src/graphql/event.rs

```diff
@@ -1298,7 +1298,7 @@ mod tests {
                         hostname: "collector1",
                         agents: [{
                             key: "piglet"
```

Comment:
I think it would be better to change this to the new nickname. This way, we can completely avoid mentioning the previous names.

Reply:
All keys have been updated.

For consistency, also updated:
learner => unsupervised
models => semi_supervised
features => sensor